### PR TITLE
Added FAQs page

### PR DESCRIPTION
**Pull request by catherinepope**
# Description
Created a FAQ page for CircleCI Academy

# Reasons
CircleCI Academy will be down for maintenance for 3 days. During that time, the URL will redirect to this page.

### CODE REVIEW

File: jekyll/_cci2/cci-academy-faqs.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: classic-docs
+title: CCI Academy FAQs
```
Comment (rosieyohannan) [inline jekyll/_cci2/cci-academy-faqs.md:3]: I recommend expanding CCI to CircleCI throughout

File: jekyll/_cci2/cci-academy-faqs.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+description: FAQs for CCI Academy
+---
+
+## CCI Academy
+
+### How do I view my registered courses?
+
+Go to your profile by clicking on your profile avatar in the upper right- hand corner of the page. You should see a list of both registered and completed courses at the bottom of the page.
+
+### What if I finished a course, but it doesn’t show as completed?
```
Comment (rosieyohannan) [inline jekyll/_cci2/cci-academy-faqs.md:13]:
```suggestion
### What if I finished a course, but it does not show as completed?
```

File: jekyll/_cci2/cci-academy-faqs.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Please [submit a ticket here](https://www.surveymonkey.com/r/ccibug) to our team about the issue. We will work on getting that fixed as soon as possible.
+
+### How can I give feedback on CCI Academy or request a course?
+
+Thanks for being willing to help us improve our website. You can [submit general feedback and course requests here](https://www.surveymonkey.com/r/DZ5ML9S).
+
+## CCI Application
+
+For questions regarding our app, please check out our [docs page]({{site.baseurl}}/2.0/faq/).
```
Comment (rosieyohannan) [inline jekyll/_cci2/cci-academy-faqs.md:27]:
```suggestion
For questions regarding the CircleCI web app, please check out our [general FAQ docs page]({{site.baseurl}}/2.0/faq/).
```

**Review by rosieyohannan — COMMENTED** [review]
Just a few suggestions. Is this page intended to only exist for 3 days, or will it be hanging around longer? If the latter could you convert to asciidoc?

File: jekyll/_cci2/cci-academy-faqs.md (hunk 1 above)
Comment (catherinepope) [reply jekyll/_cci2/cci-academy-faqs.md]: Done!

**Comment by catherinepope** [comment]
It's just for 3 days, so I did it quickly in Markdown.

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**